lib/l10n-en_GB.js

```diff
@@ -362,6 +362,10 @@ exports.messages = {
         'This date found on a boilerplate section of the document does not have the expected format (<code>DD Month YYYY</code>, with an optional leading zero in the day): <em>&ldquo;${text}&rdquo;</em>.',
     // heuristic/shortname
     'heuristic.shortname.latest-link': "Couldn't find the latest version link",
+    'heuristic.shortname.error-request':
+        'Request for TR page error, please try again or report a BUG.',
```

Comment:
The error description is pretty vague. I suggest something like
```suggestion
        'Request to ${shortlink} failed with error: ${error}',
```
and passing the right parameters.